DEVELOPMENT.md

```diff
@@ -304,7 +304,7 @@ Use relative linking to connect information between docs and jupyter notebooks,
     Here you have more control over the text that is displayed to display the module name.  When referring to a function that is alternatively defined in other modules as well, always use this option to be more explicit about which module you are referencing.
 - Link a tutorial (rst file) from within a source code docstring or rst file: ``:ref:`tutorial_name <tutorial_name>` ``
 - Link a tutorial notebook (ipynb file) from within a source code docstring or rst file: `` `notebook_name <tutorials/notebook_name.ipynb>`_ `` . (If the notebook is not the in the same folder as the source code, use a relative path)
-- Link a function from within a tutorial notebook: `[function_name](../cleanlab/file.rst#cleanlab.file.function_name)`
+- Link a function from within a tutorial notebook: `[function_name](../cleanlab/file.html#cleanlab.file.function_name)`
```

Comment:
can you clarify this'll work across both stable/ and master/ branch versions of docs?

Reply:
Added a note!

Reply:
I haven't verified it, but AIUI, the relative paths are relative to the configuration directory (the one with `conf.py`).


The relative links work across the branches.